src/Microsoft.AspNetCore.ResponseCompression/ResponseCompressionExtensions.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.Options;
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.AspNetCore.ResponseCompression
+{
+    public static class ResponseCompressionExtensions
+    {
+        /// <summary>
+        /// Allows to compress HTTP Responses.
+        /// </summary>
+        /// <param name="builder">The <see cref="IApplicationBuilder"/> instance this method extends.</param>
+        /// <param name="mimeTypes">The accepted MIME types.</param>
+        public static IApplicationBuilder UseResponseCompression(this IApplicationBuilder builder, IEnumerable<string> mimeTypes)
```

Comment:
I suppose that most users will let all other parameters as default. Maybe UseResponseCompression(this IApplicationBuilder, params string[] mimeTypes)?
Or you prefer an overload without parameter, and a default MIME type list?